src/examples/spaceshooter/renderer/WallModels.tsx

```diff
@@ -0,0 +1,112 @@
+import { Clone, useGLTF } from '@react-three/drei';
+import { memo } from 'react';
+import { Vector3 } from 'three';
+import wall_cornerMediumGLTF from '../assets/wall_cornerMedium2.glb?url';
+import wall_largeGLTF from '../assets/wall_large2.glb?url';
+import wall_mediumGLTF from '../assets/wall_medium2.glb?url';
+import levelData from '../levels/level_1';
+import { assetPath } from '../utils/RenderUtils';
+
+const WallLarge = ({
+    position,
+    rotation,
+}: {
+    position: Vector3;
+    rotation: number;
+}) => {
+    const { scene: wallLarge } = useGLTF(assetPath(wall_largeGLTF));
+    return (
+        <Clone
+            object={wallLarge}
+            position={[position.x, position.y, 0]}
+            rotation={[0, 0, rotation]}
+            scale={[10, 10, 10]}
+            deep
+        />
+    );
+};
```

Comment:
can we put each component in it's own file .... the vite dev server gets unhappy about components sharing files (and it keeps things consistent).

Reply:
Resolved: https://github.com/playmint/playerchain-demo/pull/83/commits/9d87d2fca47e743bc9fd81c8f01d7372c15502f5